Tests/UI/BillingForm/Validator/PhoneNumberValidatorTests.swift

```diff
@@ -174,7 +176,7 @@ class PhoneNumberValidatorTests: XCTestCase {
         
         XCTAssertTrue(validator.shouldAccept(text: testNumber))
         XCTAssertTrue(validator.isValid(text: testNumber))
-        XCTAssertEqual(validator.formatForDisplay(text: testNumber), "+64 9-888 8625")
+        XCTAssertEqual(validator.formatForDisplay(text: testNumber), "+64 9 888 8625")
```

Comment:
is this intentional?

Reply:
This test is dependent on the formatter, which is dependent on PhoneNumberKit. Since we have unblocked the strict linking to version 3.3.3, it would appear the later versions have changed the string representation of this country's phone number, which in turn invalidated our unit tests.

So it is intentional as I want the unit tests to pass, but I did not intentionally make changes to cause it (beyond unblocking PhoneNumberKit dependency).